polyglot/piranha/test-resources/java/non_seed_user_rule/expected/SomeClass.java

```diff
@@ -12,9 +12,9 @@
  * limitations under the License.
  */
 package com.uber.piranha;
-
-import java.util.ArrayList;
 import java.util.Collections;
+import java.util.ArrayList;
```

Comment:
Any reason we need these imports in non-alphabetical order (not blocking at all, just curious)

Reply:
So the way I have the rules, a new import statement is added at the head of the list. 
GJF reorders it, which is fine but then our test cases wont pass. 
Since our testing logic is not immune to reordering imports. 
Like if output produced by piranha is : 
```
import java.util.Collections;
import java.util.ArrayList;
```

and expected output is 
```
import java.util.ArrayList;
import java.util.Collections;
```
our test case will fail because our testing logic to account for the fact that order of import does not matter.

Reply:
Ok, makes sense. A bit unfortunate, since it means we can't just keep running GJF automatically on these files, but not worth the effort involved in addressing it right now, I think.